### DISCUSSION TITLE
图，表，公式前后各空一行的问题

### DISCUSSION THREAD

**Discussion by lei1320719647** [note286/xduts#111] [Q&A] [answered]
您好，问一下图，表，公式前后各空一行应该怎么设置呢

**Comment by note286** [comment, marked as answer]
[floats - How to change the spacing between figures/tables and text? - TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/26521/how-to-change-the-spacing-between-figures-tables-and-text)
